packages/eas-cli/src/build/local.ts

```diff
@@ -55,19 +55,26 @@ export async function runLocalBuildAsync(
   };
   process.on('SIGINT', interruptHandler);
   try {
+    const mergedEnv = {
+      ...env,
+      ...process.env,
+      EAS_LOCAL_BUILD_WORKINGDIR: options.workingdir ?? process.env.EAS_LOCAL_BUILD_WORKINGDIR,
+      ...(options.skipCleanup || options.skipNativeBuild
+        ? { EAS_LOCAL_BUILD_SKIP_CLEANUP: '1' }
+        : {}),
+      ...(options.skipNativeBuild ? { EAS_LOCAL_BUILD_SKIP_NATIVE_BUILD: '1' } : {}),
+      ...(options.artifactsDir ? { EAS_LOCAL_BUILD_ARTIFACTS_DIR: options.artifactsDir } : {}),
+      ...(options.artifactPath ? { EAS_LOCAL_BUILD_ARTIFACT_PATH: options.artifactPath } : {}),
+    };
+    // log command execution to assist in debugging local builds
+    console.log({
+      command,
+      args,
+      env: mergedEnv
+    });
```

Comment:
```suggestion
    // log command execution to assist in debugging local builds
    Log.debug('Running local build, using local-build-plugin', {
      command,
      args,
      env: mergedEnv
    });
```
and add `import Log from '../log';`

This way it will be visible only when running with `EXPO_DEBUG=1` as for other debug logs.